### PR TITLE
Feat/condominum module first

### PR DESCRIPTION
**Pull request by diegomartiiins**

### CODE REVIEW

File: back/src/application/condominium/condominum.service.ts
Comment (mathmartins2) [inline back/src/application/condominium/condominum.service.ts:1]: createCondominium can be inline. in 10 & 11

Comment (mathmartins2) [reply back/src/application/condominium/condominum.service.ts:1]: i think is better be just  `await this.findById(condominium.id);` and can remove `const { id } = condominium;`

File: back/src/application/condominium/dto/create-condominium.dto.ts
Comment (mathmartins2) [inline back/src/application/condominium/dto/create-condominium.dto.ts:1]: password length min needs to be 8

File: back/src/application/condominium/dto/update-condominium.dto.ts
Comment (mathmartins2) [inline back/src/application/condominium/dto/update-condominium.dto.ts:1]: password length min needs to be 8

File: back/src/application/condominium/errors/condominium-not-found.ts
Comment (mathmartins2) [inline back/src/application/condominium/errors/condominium-not-found.ts:1]: **_Resident not found_**  needs to be _**Condominium Not Found**_

File: back/src/application/condominium/repository/condominium.repository.ts
Comment (mathmartins2) [inline back/src/application/condominium/repository/condominium.repository.ts:1]: create return needs to be inline

Comment (mathmartins2) [reply back/src/application/condominium/repository/condominium.repository.ts:1]: implement the rest of the methods

**Review by mathmartins2 — CHANGES_REQUESTED** [review]

File: back/src/application/condominium/repository/condominium.repository.ts
Comment (diegomartiiins) [reply back/src/application/condominium/repository/condominium.repository.ts:1]: the rest of the methods will be implemented in the next commits.

**Comment by diegomartiiins** [comment]
tweaks to reviews made in the branch

**Review by mathmartins2 — APPROVED** [review]

**Pull request merged**